gradio/components/video.py

```diff
@@ -120,6 +121,7 @@ def __init__(
             min_length: The minimum length of video (in seconds) that the user can pass into the prediction function. If None, there is no minimum length.
             max_length: The maximum length of video (in seconds) that the user can pass into the prediction function. If None, there is no maximum length.
             loop: If True, the video will loop when it reaches the end and continue playing from the beginning.
+            watermark_file: An image file to be included as a watermark on the video.
```

Comment:
Let's add a description of how the watermark appears in the final video. Also if we are restricting images to specific file types, would be good to let users know here